src/components/ItaliaTheme/Blocks/Listing/Commons/ListingImage.jsx

```diff
@@ -30,9 +31,9 @@ const ListingImage = ({
   // https://stackoverflow.com/questions/33136399/is-there-a-way-to-tell-if-reactelement-renders-null
 
   const image = (
-    <a href={item['@id']} rel="noopener noreferrer">
+    <UniversalLink href={item['@id']} rel="noopener noreferrer">
```

Comment:
```suggestion
    <UniversalLink item={item}>
```

Questo rel adesso lo puoi togliere, UniversalLink è furbo abbastanza da metterlo quando serve. In aggiunta, se passi direttamente l'oggetto a cui far puntare il link, e puoi farlo essendo un blocco listing, UniversalLink fa qualche controllo e qualche magia in più.